thetis/options.py

```diff
@@ -543,8 +543,10 @@ class ModelOptions2d(CommonModelOptions):
 
         Prints deviation from the initial mass to stdout.
         """).tag(config=True)
-    tracer_advective_velocity = FiredrakeVectorExpression(None, allow_none=True,
-                                                          help="Custom function to be used for the velocity variable in tracer advection equation").tag(config=True)
+    tracer_advective_velocity_factor = FiredrakeScalarExpression(
+        Constant(1.0), help="""
+        Custom factor which is multiplied to the velocity variable in tracer advection equation
+        """).tag(config=True)
```

Comment:
Maybe you could include a sentence on when it might be useful to use this parameter?

Reply:
If the help string is long it's a good idea to split it into a short description + new line  + long description. E.g., in here https://github.com/thetisproject/thetis/blob/master/thetis/options.py#L405

Reply:
I have updated this with your suggestion